app/components/observation-form/component.js

```diff
@@ -18,6 +18,7 @@ export default class ObservationFormComponent extends Component {
   @tracked notes = '';
   @tracked isShowMapModal = false;
   @tracked isShowAddObservationModal = false;
+  @tracked currentDay = new Date();
```

Comment:
Consider to move this set into `constructor`, avoid set 'computed' properties in this way
```suggestion
  @tracked currentDay;
  
  constructor() {
    super(...arguments)
    this.currentDay = new Date()
  }
```